job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/DatabasePoller.java

```diff
@@ -84,7 +84,11 @@ public static void pollDatabaseForJobsToRun() {
                     try {
                         queueServices = QueueServicesFactory.create(jtd.getTaskPipe(),codec);
                         LOG.debug(MessageFormat.format("Sending task data to the target queue {0} ...", workerAction.toString()));
+                        if (jtd.getJobId().contains("xla")) {
+                            throw new RuntimeException("Rory testing SCMOD-10792");
+                        }
```

Comment:
What on Earth is this in our actual code? 🤯

Reply:
just for testing, to be removed on merge of course

Reply:
Hi @xbreizh, I can see that you forgot to remove this and attempted to fix it with #205.  In case you're not aware your #205 commit has not removed it (see 99f546c - there are no changes).  This is because you reused the same branch for #205 instead of creating a new one after the merge of #204.  You should always delete your local branch after it is merged and not re-use it.  Anyway I just wanted to let you know in case you missed it that the #205 commit was ineffectual.